queries.sql

```diff
@@ -8,3 +8,55 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered > 0;
 SELECT * from animals WHERE name = 'Gabumon';
 SELECT * from animals WHERE 10.4 <= weight_kg <= 17.3;
+
+/*ADD a column and 2nd. pt. Queries*/
+
+ALTER TABLE animals ADD species varchar(100);
+
+BEGIN;
+
+UPDATE animals SET species = 'Digimon' WHERE name = 'Agumon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Gabumon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Devimon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Plantmon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Angemon';
+UPDATE animals SET species = 'Digimon' WHERE name = 'Boarmon';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Pikachu';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Charmander';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Squirtle';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Blossom';
+UPDATE animals SET species = 'Pokemon' WHERE name = 'Ditto';
+
+ROLLBACK;
+
+BEGIN;
+
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'Pokemon' WHERE species = NULL;
+
+COMMIT;
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+BEGIN;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT SP1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
+
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT name FROM animals WHERE (escape_attempts = (SELECT MAX (escape_attempts) FROM animals)) OR (escape_attempts = 1) OR (escape_attempts = 0);
```

Comment:
- [ ] Kindly correct this query. The query should return the animal that has escaped the most whether the animal is neutered or not. To do that, you can remove the `OR (escape_attempts = 1) OR (escape_attempts = 0)`

Reply:
done